### ISSUE TITLE
Initial support of debugging

### ISSUE THREAD

**Issue by daimor** [intersystems-community/vscode-objectscript#2]

**Comment by daimor** [comment]
some work in progress e02a91f1aad51f402c0f743eb6535bb6f7f96b23

**Comment by daimor** [comment]
@david2278, just pushed new changes to support attach to process.
the latest build should appear in [releases](https://github.com/daimor/vscode-objectscript/releases)

**Comment by daimor** [comment]
```JSON
{
  "version": "0.2.0",
  "configurations": [    
    {
      "type": "objectscript",
      "request": "launch",
      "name": "ObjectScript Debug Class",
      "program": "##class(User.Test).Test()",
    },
    {
      "type": "objectscript",
      "request": "launch",
      "name": "ObjectScript Debug Routine",
      "program": "^test",
    },
    {
      "type": "objectscript",
      "request": "attach",
      "name": "ObjectScript Attach",
      "processId": "${command:PickProcess}",
      "system": true
    }
  ]
}
```

**Comment by david2278** [comment]
Awesome thanks!!! I will be testing it today.

**Comment by rfns** [comment]
Hey, nice to see something going for the debugger. However it seems it's still not available on my Debug list. Do you think there is something missing in the package.json to make it visible?

**Comment by rfns** [comment]
Looks like this debug type is not supported.

`"type": "objectscript",`

**Comment by daimor** [comment]
If it did not underline "objectscript" with yellow, it should work, then.

**Comment by rfns** [comment]
It did though... I can't run the debug because a message appears right after I try running it, this message  is about the same as the one showing with the yellow line.

**Comment by daimor** [comment]
So, just check if you use the latest version, should be 0.7.12
![image](https://user-images.githubusercontent.com/1212251/64049199-b84dcb00-cb7c-11e9-9c54-246222624275.png)
You can also choose which version to install by clicking on gear

**Comment by rfns** [comment]
Hmm, yeah, it seems I was still using the version 0.7.11, let me try again.

**Comment by rfns** [comment]
Yeah, I'm not crazy, I'll try posting a gif to show you what's happening...

https://i.imgur.com/lozkkBB.gif

**Comment by daimor** [comment]
wow, so, interesting

**Comment by daimor** [comment]
well, how about, your version of VSCode itself?

**Comment by rfns** [comment]
Yeah, I just noticed that you upgraded the vscode to a more recent version, this is the why I was stuck at 0.7.11, hold on, now it should work.

Update: Yeah, it works, I can run it but I can't set any breakpoints on cls files. The debugger runs, executes successfuly and quits immediately. You can see how I configured it in the gif from earlier.

**Comment by daimor** [comment]
with setting breakpoints possible some issues with not so correct detecting file location
you can at least add `Break` command, to check how it works

**Comment by daimor** [comment]
It is also possible, that some issues because of Windows, as I use macOS

**Comment by mat1990dj** [comment]
I have a question, how can I pass parameters to my routine? My debug config is similar to:
 {
      "type": "objectscript",
      "request": "launch",
      "name": "ObjectScript Debug Class",
      "program": "##class(User.Test).Test("a", "b")",
    },
